ale_linters/sh/language_server.vim

```diff
@@ -0,0 +1,34 @@
+" Author: Christian Höltje (https://docwhat.org/)
+" Description: BASH Language server integration for ALE
+scriptencoding utf-8
+
+call ale#Set('sh_language_server_executable', 'bash-language-server')
+call ale#Set('sh_language_server_use_global', get(g:, 'ale_use_global_executables', 0))
+
+function! ale_linters#sh#language_server#GetExecutable(buffer) abort
+    return ale#node#FindExecutable(a:buffer, 'sh_language_server', [
+    \   'node_modules/.bin/bash-language-server',
+    \   'bash-language-server',
```

Comment:
You can remove this line here. These strings are for paths to search upwards for. The global executable will be used if a local one can't be found.

Reply:
done